osu.Game/Rulesets/RealmRulesetStore.cs

```diff
@@ -143,5 +149,41 @@ private void testRulesetCompatibility(RulesetInfo rulesetInfo)
 
             instance.CreateBeatmapProcessor(converter.Convert());
         }
+
+        private void informUserAboutBrokenRulesets()
+        {
+            if (RulesetStorage == null)
+                return;
+
+            foreach (string brokenRulesetDll in RulesetStorage.GetFiles(@".", @"*.dll.broken"))
+            {
+                Logger.Log($"Ruleset '{Path.GetFileNameWithoutExtension(brokenRulesetDll)}' has been disabled due to causing a crash.\n\n"
+                           + "Please update the ruleset or report the issue to the developers of the ruleset if no updates are available.", level: LogLevel.Important);
+            }
+        }
+
+        internal void TryDisableCustomRulesetsCausing(Exception exception)
```

Comment:
On a quick check of methods called, it doesn't look like this could through.. but even so a `try-catch` would set my mind at ease, especially given them method name.